recipes/gaia/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
     - celery >=3.1.20
     - psycopg2 >=2.6.1
     - geoalchemy2 >=0.2.6
+    - rasterio
```

Comment:
I wasn't aware of this package.  It seems it comes from the mapbox people and has a lot of overlap with what gdal aims to provide.  Reading their "Philosophy", it seems they want to do what GDAL does, but in a more expressive and Pythonic way.  Their docs just say they depend on gdal >= 1.9, so I guess we should be ok here.

+1